policies/modules/ROOT/pages/policies-automated-overview.adoc

```diff
@@ -2,19 +2,19 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-aliases: api-manager::automated-policies-landing-page.adoc
+// :page-aliases: api-manager::automated-policies-landing-page.adoc
 
-Configuring and using automated policies enables your Anypoint Platform instance to comply with common security and logging requirements by automatically applying the same set of policies to all APIs running in a single environment. With policy automation, API owners can quickly design, build, and deploy secure and consistent APIs.
+Configuring and using automated policies enables your API instance to comply with common security and logging requirements by automatically applying the same set of policies to all APIs running in Flex Gateway, or in Mule. With policy automation, API owners can quickly design, build, and deploy secure and consistent APIs.
 
 With automated policies, API owners can deploy their APIs with confidence because the target environment automatically enforces critical polices. This ensures consistency and reduces possible errors. An administrator no longer needs to remember to insert a policy to enforce common standards manually.
 
 Because best practices are built into the application network, API owners can design, build, and deploy APIs quickly without compromising security.
 
-All of the xref:policies-included-directory.adoc[included policies], and any xref:policies::policies-custom-overview.adoc[custom policy] can be configured as automated policies and can be configured to be applied at all the API’s running under an environment.
+All of the xref:policies-included-directory.adoc[included policies], and any xref:policies::policies-custom-overview.adoc[custom policy] can be configured as automated policies and can be applied at all the API’s running under an environment.
 
-== Support for Automated Policies
+== Automated Policies in Mule
 
-The following rules apply to automated policies:
+The following rules apply to automated policies in Mule:
 
 * Automated policies are only available for Mule runtime engine (Mule) 4 APIs using a MuleSoft-hosted control plane.
```

Comment:
Usually, the first instance of Mule has the expanded name.

Reply:
Fixed